.golangci.yml

```diff
@@ -30,104 +26,103 @@ linters:
     - prealloc
     - revive
     - staticcheck
-    - stylecheck
```

Comment:
stylecheck typecheck as well

Reply:
stylecheck has been merged into staticcheck
typecheck is now forced